.github/workflows/doctests.yml

```diff
@@ -0,0 +1,33 @@
+name: doctests
+on:
+  push:
+    branches:
+      - 'main'
+      - /^release-.*$/
+    tags: '*'
+  pull_request:
+jobs:
+  doctests:
+    name: Doctests (Julia ${{ matrix.julia-version }} - ${{ github.event_name }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        julia-version: [1.6.1]
```

Comment:
I feel like for doctests we can just omit the matrix, I doubt we'd ever want to run them on more than one version. We can just put the `1.6.1` directly in the `setup-julia` action.

Reply:
Yeah, true. My reason for keeping the matrix was so that I could put the version in the job name, so there’s more visibility on what version of Julia is being used by the doctests.

Is there another way to do that? I am also worried about it getting out of sync if we just write it in both places

Reply:
It seems a little odd to specify the exact version instead of just 1, since then the doctests could get behind what people would see on a fresh install of the latest stable version